### ISSUE TITLE
[BUG] t.filter is not a function

### ISSUE THREAD

**Issue by karmiller** [Chia-Network/chia-blockchain-gui#1019]
### What happened?

Hi I have 3 NFTs within the same collect that that every time i click the GUI errors with the below and the wallet wants me to reload the app.    Something went wrong
Error: t.filter is not a function
at webpack://@chia/gui/src/components/nfts/NFTRankings.tsx:70:23 NFTRankings
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:172:114 useLayoutEffect
at webpack://@chia/gui/node_modules/react/cjs/react.production.min.js:23:112 useLayoutEffect
at webpack://@chia/gui/src/components/nfts/NFTRankings.tsx:69:29 NFTRankings
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:157:136 Ch
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:267:459 b?b.sibling:null}I
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:250:346 Pj
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:250:277 Pj
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:250:137 Pj
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:243:162 Lj

### Version

15.1 but tried in 14 and 15

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output or stacktrace

```shell
t.filter is not a function

URL
#/dashboard/nfts/nft1h574vc6ujs3luu5yn3670httdwep2nqkz4zuk2qadv8ge5ppgqrqguejmc

Stacktrace
at webpack://@chia/gui/src/components/nfts/NFTRankings.tsx:70:23 NFTRankings
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:172:114 useLayoutEffect
at webpack://@chia/gui/node_modules/react/cjs/react.production.min.js:23:112 useLayoutEffect
at webpack://@chia/gui/src/components/nfts/NFTRankings.tsx:69:29 NFTRankings
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:157:136 Ch
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:267:459 b?b.sibling:null}I
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:250:346 Pj
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:250:277 Pj
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:250:137 Pj
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:243:162 Lj
```

**Comment by paninaro** [comment]
Thanks for the report @karmiller. I can repro the issue.

**Comment by karmiller** [comment]
hi. does that mean it should work now. ive download 1.6 and it still happens

**Comment by paninaro** [comment]
@karmiller the fix will be in 1.6.1. Sorry for the confusion, the issue was closed by an automated workflow because the fix was merged into the main branch.
